signals/generators/ios/ios_generator.py

```diff
@@ -75,6 +78,26 @@ def copy_data_models(self):
         shutil.copyfile(self.header_file, "{}/DataModel.h".format(self.data_models_path))
         shutil.copyfile(self.implementation_file, "{}/DataModel.m".format(self.data_models_path))
 
+    def check_setup_called(self):
+        # Define pattern we're looking for to verify that setup was called
+        pattern = re.compile("sharedDataModel\(\)\.setup\(\)")
+
+        # Find parent path that contains AppDelegate.swift
+        # TODO: Update this to include AppDelegate.m once an objective-c template exists
+        parent_path_containing_target = recursively_find_parent_containing_file(self.data_models_path,
+                                                                                ["AppDelegate.swift"])
+        if parent_path_containing_target is None:
+            warn("Warning: Unable to find AppDelegate.swift to verify sharedDataModel().setup() is called")
+            return False
+
+        # Walk the file looking for setup call
+        for i, line in enumerate(open(parent_path_containing_target + os.sep + "AppDelegate.swift")):
+            for match in re.finditer(pattern, line):
```

Comment:
Agreed, this is maybe overkill for our situation.  But I could imagine some people wanting to support different coding styles, such as <code>setup( )</code> or <code>setup ()</code>, at which point it would be trivial to include that variation in the regex.  Should we favor simplicity at this point?